head_tp_controller:
  type: "tiago_telepresence_controllers/HeadControllerPlugin"
  joint_names:
  - head_1_joint
  - head_2_joint
  buffer_min_size: 5

torso_tp_controller:
  type: "tiago_telepresence_controllers/TorsoControllerPlugin"
  joint_names:
  - torso_lift_joint
  step: 0.005

gripper_tp_controller:
  type: "tiago_telepresence_controllers/GripperControllerPlugin"
  joint_names:
  - gripper_left_finger_joint
  - gripper_right_finger_joint
  step: 0.005

arm_tp_controller:
  type: "tiago_telepresence_controllers/ArmControllerPlugin"
  joint_names:
  - arm_1_joint
  - arm_2_joint
  - arm_3_joint
  - arm_4_joint
  - arm_5_joint
  - arm_6_joint
  - arm_7_joint
  start_link: "torso_lift_link"
  end_link: "gripper_grasping_frame"
  buffer_min_size: 5
  state_publish_throttle: 0.05
